src/components/MessagesList/MessagesList.vue

```diff
@@ -1106,66 +1094,35 @@ export default {
 		},
 
 		/**
-		 * @param {object} options Event options
-		 * @param {boolean} options.force Set to true, if the chat should be scrolled to the bottom even when it was not before
-		 */
-		handleScrollChatToBottomEvent(options) {
-			if ((options && options.force) || this.isChatScrolledToBottom) {
-				this.scrollToBottom()
-			}
-		},
-
-		/**
-		 * Scrolls to the bottom of the list (to show reaction to the last message).
+		 * Scrolls to the bottom of the list.
+		 * @param options {object} Options for scrolling (smooth: boolean, force: boolean)
```

Comment:
```suggestion
		 * @param {object} options Options for scrolling
		 * @param {boolean} [options.smooth] 'smooth' scrolling to the bottom ('auto' by default)
		 * @param {boolean} [options.force] force scrolling to the bottom (otherwise check for current position)
```